sqlair.go

```diff
@@ -25,7 +28,14 @@ var ErrNoRows = sql.ErrNoRows
 // Statement represents a SQL statement with valid SQLair expressions.
 // It is ready to be run on a SQLair DB.
 type Statement struct {
+	id uuid.UUID
```

Comment:
Can we please call this as `cacheID` so it's more obvious why we have it?

Reply:
Changed.